docs/content/gameobject/sending-receiving-commands.md

```diff
@@ -128,7 +132,7 @@ public class SpawnCubeCommandRequestHandlerBehaviour : MonoBehaviour
 ```
 [//]: # (TODO - add how to do spawn logic in code example above)
 
-**Note:** For an example of how to create an entity, you can look at the [how to create and delete entities](TODO: GDoc 26) document.
+> For an example of how to create an entity, you can look at the [how to create and delete entities]({{urlRoot}}/content/gameobject/create-delete-spos-entries) document.
```

Comment:
Are you sure the URL is correct?

Reply:
Hilariously - it is. There's a typo in the filename

Reply:
🤦‍♂️